tendrl/gluster_integration/objects/geo_replication_session/__init__.py

```diff
@@ -0,0 +1,35 @@
+from tendrl.commons import objects
+
+
+class GeoReplicationSessionStatus(object):
+    ACTIVE = "active"
+    PARTIAL = "partial"
+    FAULTY = "faulty"
+
+
+class GeoReplicationSession(objects.BaseObject):
+    def __init__(
+        self,
+        vol_id,
+        session_id,
+        session_status,
+        pairs=None,
+        *args,
+        **kwargs
+    ):
+        super(GeoReplicationSession, self).__init__(*args, **kwargs)
+
+        self.vol_id = vol_id
+        self.session_id = session_id
+        self.session_status = session_status
+        if pairs:
```

Comment:
yes, Since pairs is a different object object. It normally gets updated separately hence the condition